.github/workflows/chain_interaction.yml

```
@@ -9,39 +9,54 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout sources
+      - name: Checkout 🛎️
         uses: actions/checkout@v2
 
-      - name: Install stable toolchain
+      - name: Verify .rs and toml files 👀
+        uses: technote-space/get-diff-action@v6.1.0
+        id: git_diff
+        with:
+          PATTERNS: |
+            **/**.rs
+            **/**.toml
```

```suggestion
          PATTERNS: |
            **/**.rs
            **/**.toml
            **/Cargo.lock
          FILES: |
            Cargo.lock
```
 `Cargo.lock` should be included.